examples/dart/bin/myapp.dart

```diff
@@ -15,6 +15,35 @@ void main(List<String> arguments) {
   print("Bundled realm location: " + realm.config.path);
   realm.close();
   // :snippet-end:
+  Future<void> createSyncedBundle() async {
+    final APP_ID = '';
+    // :snippet-start: create-synced-bundle
+    print("Bundling synced realm");
+
+    // You must connect to the Device Sync server with an authenticated
+    // user to work with the synced realm.
+    final app = App(AppConfiguration(APP_ID));
+    final user = await app.logIn(Credentials.anonymous());
```

Comment:
```suggestion
    final user = app.currentUser ?? await app.logIn(Credentials.anonymous());
```
Once you have logged in, you don't need to again.. holds for anonymous login as well